server/push/tnpg/push_tnpg.go

```diff
@@ -26,6 +26,7 @@ type configType struct {
 	Enabled          bool `json:"enabled"`
 	Buffer           int  `json:"buffer"`
 	CompressPayloads bool `json:"compress_payloads"`
+	BatchSize        int  `json:"batch_size"`
```

Comment:
FCM SendAll supports batches of 100 messages and that's it:
https://godoc.org/firebase.google.com/go/messaging#Client.SendAll

Configuring more than 100 won't work. Configuring fewer is just wasteful. I see no point in letting users configure this value if FCM does not let us configure it. A hardcoded 100 seems like the best choice.

Reply:
Sure.